.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+AllCops:
+  TargetRubyVersion: 2.3
+
+Style/FrozenStringLiteralComment:
```

Comment:
It's there because when I run `bundle exec govuk-lint-ruby` locally it fails without it. How should I ensure the other-style.yml is being used?